tests/testthat/_snaps/cnd-abort.md

```diff
@@ -1162,7 +1162,7 @@
 # must pass character `body` when `message` is > 1
 
     Code
-      err(abort("foo", body = function(cnd) c(i = "bar")))
+      err(abort("foo", body = function(cnd, ...) c(i = "bar")))
```

Comment:
yes but I was going to wait for revdep checks to see if we need any defensive measure against this change.

That said I can also revert it because I switched from passing an argument to defining a special cli theme.